pkg/config/config.go

```diff
@@ -176,10 +176,11 @@ type (
 	FunctionConfig map[string]function
 
 	function struct {
-		Enabled    *bool  `toml:"enabled" json:"-"`
-		VerifyJWT  *bool  `toml:"verify_jwt" json:"verifyJWT"`
-		ImportMap  string `toml:"import_map" json:"importMapPath,omitempty"`
-		Entrypoint string `toml:"entrypoint" json:"entrypointPath,omitempty"`
+		Enabled            *bool   `toml:"enabled" json:"-"`
+		VerifyJWT          *bool   `toml:"verify_jwt" json:"verifyJWT"`
+		ImportMap          string  `toml:"import_map" json:"importMapPath,omitempty"`
+		Entrypoint         string  `toml:"entrypoint" json:"entrypointPath,omitempty"`
+		ResourceMultiplier *string `toml:"resource_multiplier" json:"resourceMultiplier,omitempty"`
```

Comment:
Why is this a string and not float?

https://github.com/supabase/cli/pull/2877/files#diff-44113b100b5a3840b42b5d08768b70a8f82055f9a7d11b37f31e2ea72d51e50dR152